### ISSUE TITLE
How to import existing cluster for `talos_machine_bootstrap` resource?

### ISSUE THREAD

**Issue by james-callahan** [siderolabs/terraform-provider-talos#40]
To import an existing talos cluster into terraform, how can I make the `bootstrap` resource happy?

Currently if you try (e.g. with an empty import string) you get:
```
│ Error: resource talos_machine_bootstrap doesn't support import
```

**Comment by frezbo** [comment]
i think this is more tricky and I'm not sure if we want to add all the checks for this into the provider yet.

**Comment by james-callahan** [comment]
As an MVP you could just support an empty import string: it's not as if you save any state from the bootstrap in the terraform state is it?
https://github.com/siderolabs/terraform-provider-talos/blob/158dbbde42326fed9ada2a97a2a25c625c0a783e/talos/resource_talos_machine_bootstrap.go#L66

**Comment by frezbo** [comment]
that's an easy hack, it should work for now.

**Comment by frezbo** [comment]
This is fixed in v1.4.0-alpha.0
